src/schema/v2/me/conversation/index.ts

```diff
@@ -352,20 +364,41 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
     },
 
     items: {
+      args: {
+        live: {
+          type: GraphQLBoolean,
+          description: "When true fetches an actual items instead of snapshots",
+          defaultValue: false,
+        },
+      },
       type: new GraphQLList(ConversationItem),
       description:
         "The artworks and/or partner shows discussed in the conversation.",
-      resolve: (conversation) => {
+      resolve: async (conversation, args, { artworkLoader }, _info) => {
         const results = []
 
         for (const item of conversation.items) {
           if (
             isExisty(item.properties) &&
             (item.item_type === "Artwork" || item.item_type === "PartnerShow")
           ) {
-            // FIXME: Argument of type 'any' is not assignable to parameter of type 'never'.
-            // @ts-ignore
-            results.push(item)
+            if (args.live && item.item_type === "Artwork") {
+              const artwork = await artworkLoader(item.properties.id)
+
+              const updatedArtwork = {
+                properties: artwork,
+                item_type: "Artwork",
+                title: artwork.title,
+                permalink: `https://www.artsy.net/artwork/${item.properties.id}`,
+              }
```

Comment:
One thing about this solution is that we have to do some data massaging.

Reply:
This is to me the code smell that makes me in favor of `liveItem` instead. Aside from not being up to date the `item` snapshot sent from impulse has its own shape and so when fetching a real artwork we have to do extra work to conform to the resolver's conversation-item shape. For this reason I said let's just make this a separate property and call it only when specifically requested. We could also add a `description`  that `item` is only a snapshot.